tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2EvpTests.cs

```diff
@@ -124,10 +127,15 @@ public async Task SubmitTraces(string packageVersion, string evpVersionToRemove,
 
                     using (ProcessResult processResult = await RunDotnetTestSampleAndWaitForExit(agent, packageVersion: packageVersion))
                     {
+                        var settings = VerifyHelper.GetCIVisibilitySpanVerifierSettings(expectedTestCount == 15 ? "old" : "new", null, null);
```

Comment:
I would advice not using "old" and "new" unless you want "new new" and "newer" to be suffixes down the line 😛 You'd be better off with things like `Pre_2` etc

I'd also suggest you don't base it on the `expectedTestCount`, but instead have a bool/enum that controls both the number of exepected tests _and_ the suffix. Otherwise, when you add a new test later to check some functionality you're going to have to remember to update all these